helper.go

```diff
@@ -46,6 +46,9 @@ func MGetCache(client Client, ctx context.Context, ttl time.Duration, keys []str
 	if len(keys) == 0 {
 		return make(map[string]RedisMessage), nil
 	}
+	if isCacheDisabled(client) {
+		return MGet(client, ctx, keys)
```

Comment:
We need tests on this.

Reply:
Will update the tests and cover these conditions

Reply:
I have covered that case, can you please take a look again? @rueian